lib/generator/src/Generator/AbstractTypeGenerator.php

```diff
@@ -67,6 +69,11 @@ function (%s) <closureUseStatements>{
      */
     protected $cacheDirMask;
 
```

Comment:
The `$currentlyGeneratedClass` is used to know in any generator class which definition is being generated right now.